src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -42,14 +42,33 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     // But, we specifically need an API Token key-pair of Code Studio.
     // So we reauthenticate to be sure we're using the provided credentials.
     $this->reAuthenticate($cloudKey, $cloudSecret, $this->cloudCredentials->getBaseUri(), $this->cloudCredentials->getAccountsUri());
-
-    $phpVersions = [
-      'PHP_version_8.1' => "8.1",
-      'PHP_version_8.2' => "8.2",
+    $phpVersion = NULL;
+    $nodeVersion = NULL;
+    $projectType = [
+      'Drupal_project',
```

Comment:
Please kill the escaped mutants. For instance, this escaped mutant is telling you that if `Drupal_project` is removed as an available project type, tests still pass. That seems like something tests should catch..